rcp/org.testeditor.rcp4.views.teststepselection/src/org/testeditor/rcp4/views/teststepselector/TestStepSelector.xtend

```diff
@@ -76,6 +80,29 @@ class TestStepSelector {
 
 				})
 		]
+		val page = PlatformUI.getWorkbench().getActiveWorkbenchWindow().getActivePage();
```

Comment:
val page = PlatformUI.workbench.activeWorkbenchWindow.activePage

(get can be removed, () can be removed and ; too)